opal/mca/accelerator/cuda/accelerator_cuda.c

```diff
@@ -554,6 +602,12 @@ static int accelerator_cuda_get_buffer_id(int dev_id, const void *addr, opal_acc
 {
     CUresult result;
     int enable = 1;
+
+    result = opal_accelerator_cuda_delayed_init();
```

Comment:
result is a CUresult but is assigned an OPAL error code. Below, a CUDA error code is returned as OPAL error code (https://github.com/open-mpi/ompi/pull/11253/files#diff-39b9e960ab651004b6fb3f54ce4fa24ec51d4c91d9028e8a8d5bbd69ba127830R589)